src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/create_window_request.cc

```diff
@@ -333,7 +333,10 @@ bool CreateWindowRequest::DoesExceedMaxAllowedWindows(
         return false;
       });
 
-  DCHECK(find_res != windowTypeSupported->end());
+  if (find_res == windowTypeSupported->end()) {
+    LOG4CXX_WARN(logger_, "Requested Window Type is not supported by the HMI");
+    return false;
```

Comment:
```suggestion
    return true;
```
?
It seems to me that the error case would be appropriate here.